README.md

```diff
@@ -9,3 +9,34 @@ Mini sprite is a simple, matrix based format for creating 1bit styled graphics.
 [license_link]: https://opensource.org/licenses/MIT
 [very_good_analysis_badge]: https://img.shields.io/badge/style-very_good_analysis-B22C89.svg
 [very_good_analysis_link]: https://pub.dev/packages/very_good_analysis
+
+<img src="art/logo.png" width="250" />
+
+## Project structure
+
+Mini Sprite is composed by several projects:
+
+### [Mini Sprite](packages/mini_sprite)
+
+The core of the it contains the engine/platform agnostic core code to handle the format for the
+sprites and maps.
+
+### [Mini Sprite Editor](packages/mini_sprite_editor)
+
+The flutter application deployed at [https://minisprit.es](https://minisprit.es) which is used to design sprites and
+maps using Mini Sprite's format.
+
+### [Flame Mini Sprite](packages/flame_mini_sprite)
```

Comment:
Shouldn't the bridge library live in the Flame monorepo?

Reply:
It could! But I thought it is too soon for this to be a thing in the flame org.

Would love to move it though (but probably later with more time 😅 )